src/error.rs

```diff
@@ -26,6 +26,10 @@ pub enum Error {
         full_command: String,
         source: Arc<FromUtf8Error>,
     },
+    CradleBug {
```

Comment:
I slightly prefer `Internal` as the name for this error, since internal errors come up in other contexts, like http 500s, although I don't hate `CradleBug`. In that case I might call it just `Bug`, since it's already a `cradle::Error`.

```suggestion
    Internal {
```

Reply:
Done.